specification/sphere/Sphere.Management/image.tsp

```diff
@@ -71,7 +73,7 @@ model ImageProperties {
 
 @doc("Image upload request body.")
 model ImageUploadRequestBody {
```

Comment:
This model is not used anywhere. Did you intend to use it in the new POST?

Reply:
removed it in latest commit - it was not needed